mobile/test/builtin_extensions_text.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:immich_mobile/utils/builtin_extensions.dart';
+
+void main() {
+  group('Test toDuration', () {
+    test('ok', () {
+      expect(
+        "1:02:33".toDuration(),
+        const Duration(hours: 1, minutes: 2, seconds: 33),
```

Comment:
You are testing the observed behavior of the built in, but it would be better to test the actual implementation of `Asset.remote`. Can you update the test to call that function instead? And pass in a valid/invalid duration, which should _not_ throw an error, but default it to 0?

Reply:
Er, I guess testing the built in is good, but in the future it would be great to write some tests against the actual `Asset.remote` implementation since that is the behavior we actually want to validate works as expected. If the implementation changed _there_ we could have a regression, even though your extension still works as designed.

Reply:
that'd be an entirely new test case and should test much more than only this tiny piece. 
I'm testing toDuration() because it's hand written by us/me and not a library function

Reply:
Sounds good, this works for now for sure. Would definitely be worth writing a regression test for this in a subsequent PR though, so wherever that makes sense in your transition-to-offline roadmap :smile: